Makefile

```diff
@@ -11,7 +11,7 @@
 RCNUM=${RC_NUM}
 WEBROOT=${K8S_WEBROOT}
 K8SROOT=${K8S_ROOT}
-K8SRELEASE=${K8S_RELEASE}
+K8SRELEASE=${K8S_RELEASE:1.20.0}
```

Comment:
Does this substitution syntax actually work in a Makefile context?  It appears that it's trying to expand an undefined variable if I run make with `--warn-undefined-variables`:

```
Makefile:35: warning: undefined variable 'K8S_RELEASE:1.20.0'
```

Reply:
The original variable substitution worked correctly (set `K8S_RELEASE`).
I have not tested this syntax:`{K8S_RELEASE:1.20.0}`.
cc @tengqm

Reply:
looks like my fault, will confirm and fix if needed.